popup/search-results.css

```diff
@@ -0,0 +1,129 @@
+#disable-all-wrapper {
+  border-bottom: solid 1px #000;
+  margin-bottom: 5px;
+}
+
+#searchResults,
+#find-styles {
+  display: block;
+}
+
+#searchResults.hidden,
+#searchResults-error.hidden,
+#find-styles.hidden,
+#open-search.hidden,
+.searchResult-customize.hidden {
+  display: none;
+}
+
+#searchResults-error {
+  background-color: rgba(255, 0, 0, 0.4);
+  font-weight: bold;
+  padding: 5px;
+  text-align: center;
+  margin-right: 20px;
+}
+
+#searchResults, #searchResults-list {
+  max-width: 200px;
+  overflow: hidden;
+}
+
+.searchResult {
+  padding: 5px;
+}
+.searchResult:hover {
+  background-color: rgba(50, 150, 255, 0.5);
+}
+.searchResult.loading {
+  opacity: 0.5;
+}
+
+.searchResult img {
+  display: block;
+  max-width: 180px;
+  max-height: 180px;
+}
+.searchResult-screenshot.no-screenshot {
+  width: 180px;
+  height: 40px;
+}
+
+.searchResult-title {
+  display: block;
+  font-weight: bold;
+}
+
+.searchResult-description {
+  overflow: hidden;
+  font-style: italic;
+}
+
+.searchResult > * {
+  overflow-x: hidden;
+  white-space: nowrap;
```

Comment:
Hiding the overflow makes it hard to quickly rad the titles, let the title flow and add `text-overflow: ellipsis` on the description